### DISCUSSION TITLE
live_grep_glob_resume functionality?

### DISCUSSION THREAD

**Discussion by BSteffaniak** [ibhagwan/fzf-lua#1182] [Ideas]
Would it make sense to have a live_grep_glob_resume functionality?

I was able to get it working the way I intended by adding this in lua/fzf-lua/providers/grep.lua:
```
M.live_grep_glob_resume = function(opts)
  if not opts then
    opts = {}
  end
  opts.resume = true
  return M.live_grep_glob(opts)
end
```

and this in lua/fzf-lua/init.lua:
```
...
live_grep_glob_resume = { "fzf-lua.providers.grep", "live_grep_glob_resume" },
...
```

**Comment by ibhagwan** [comment]
Everything can be resumed, `live_grep_resume` is just an alias, `live_grep_glob_resume` can be replicated with `:FzfLua live_grep_glob resume=true`.

**Comment by ibhagwan** [comment]
Also, if you have `rg_glob=true` in your setup under grep, then live_grep/grep is already “glob enabled” (live_grep = live_grep_glob) and therefore `live_grep_resume` is technically `live_grep_glob_resume`.

**Comment by BSteffaniak** [comment]
That works for me. Thanks!
